src/components/LoginComponent.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import "../styles/form.style.scss"
+import {Formik} from "formik"
+import * as yup from "yup"
+import { useNavigate } from 'react-router-dom'
+
+const mySchema = yup.object().shape({
+  Email: yup.string().required("Username is requires!"),
+  Password: yup.string().min(8, "login must be at least 8 chars long!")
+})
+
+function LoginComponent() {
```

Comment:
Rename to just Login, it's already in components folder, no need to say Component in the name